file/builder.go

```diff
@@ -188,6 +188,9 @@ func (b *stateBuilder) consumerGroups() {
 				return
 			}
 		}
+		if current != nil && current.CreatedAt != nil {
+			cgo.ConsumerGroup.CreatedAt = current.CreatedAt
+		}
```

Comment:
why do we need to check `current.CreatedAt != nil`? if set to nil, kong or konnect will set it properly to now(), no?

Reply:
Of course 🤦

Reply:
@aboudreault can you take another look? https://github.com/Kong/deck/pull/1061/commits/8cd9e138161ba442ab14dabbd6177f6088c0b4a5